### DISCUSSION TITLE
GraphQL Playground

### DISCUSSION THREAD

**Discussion by leecalcote** [meshery/meshery#2348] [General]
https://github.com/layer5io/meshery/pull/2344#discussion_r569563494 @kumarabd

**Comment by leecalcote** [comment]
@kumarabd

**Reply by leecalcote** [reply]
This needs documented.

**Reply by kumarabd** [reply]
Lets close this. That has been removed from the code.

**Comment by leecalcote** [comment]
We just pulled up the playground yesterday.  Is this part of Meshery extensions?

**Reply by kumarabd** [reply]
No, the Meshery server.
